### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Add Account_Connect to Azure Storage_Screen Reader: While navigating to the "Subscription" etc. buttons NVDA is not announcing the label "What kind of Azure resource do you want to connect to?"

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/AzureStorageExplorer#7581]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11 Enterprise Insider Preview (23H2)

### GitHub Tags:
#A11yMAS;#A11yTCS;#AILimited;#Win11;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#FTP;#ScreenReader;#WCAG1.3.1;#NVDA;#Element:Button;#STP;#Narrator

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Narrator, NVDA - version 2023.1

### Steps to Reproduce

1. Turn on Narrator/NVDA
2. Launch Storage Explorer
3. Tab till "Open Connect Dialog" pane present on left side of screen and hit enter.
4. "Connect to Azure Storage" popup opens
5. Verify that While navigating to the "Subscription" etc. buttons NVDA is announcing the label "What kind of Azure resource do you want to connect to?" or not.

### Actual Experience

While navigating to the "Subscription" etc. buttons NVDA is not announcing the label "What kind of Azure resource do you want to connect to?"

### Expected Experience

When keyboard focus lands on "Subscription" button screen reader should announce the "What kind of Azure resource do you want to connect to?" label.

### User Impact:
Screen Reader users will face difficulty in accessing the controls if the complete information is not being announced by screen reader.

### Attachment:
![A11y_Azure Tools Storage Explorer_Add Account_Connect to Azure Storage_Screen Reader_Narrator](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/5cb955fb-5c45-446d-8f4d-6b24e9f6ff78)

![A11y_Azure Tools Storage Explorer_Add Account_Connect to Azure Storage_Screen Reader_NVDA](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/9d7240a8-9827-414f-89ae-71fb35f9d1c4)

**Comment by kupatkar99** [comment]
Discussed the issue in office hours call and the bug is logged with valid severity.

**Comment by Mana-swi** [comment]
#a11ydb-group-not-exposed;
